.ls-lint.yml

```diff
@@ -9,11 +9,14 @@ ls:
     .svg: regex:([.a-z0-9]*)([-.][a-z0-9]+)*
     .webp: regex:([.a-z0-9]*)([-.][a-z0-9]+)*
     .gif: regex:([.a-z0-9]*)([-.][a-z0-9]+)*
-    .ttf: regex:([.a-z0-9]*)([-.][a-z0-9]+)*
-    .woff: regex:([.a-z0-9]*)([-.][a-z0-9]+)*
-    .woff2: regex:([.a-z0-9]*)([-.][a-z0-9]+)*
     backend/src/migrations:
         .ts: snake_case
+    frontend/src/assets/fonts:
```

Comment:
this is not part of the design. It is just header block
![image](https://user-images.githubusercontent.com/78347072/225696902-c9976ff0-bc87-4b34-94f6-66e6b7c0f460.png)

Reply:
Linter cursed at capital letters in font file names. I decided that it would be faster to fix in the file, fem each file name + in the font file itself

Reply:
Regarding Rubik, there are two more fonts in the layout that are used